### PR TITLE
input-refactor: another batch of changes

### PR DESCRIPTION
**Pull request by dkochmanski**

### CODE REVIEW

**Comment by dkochmanski** [comment]
rebased on current master

File: Core/clim-basic/recording.lisp [hunk 1]
```diff
@@ -1937,57 +1934,57 @@ add output recording facilities. It is not instantiable."))
...
-(defmethod stream-add-character-output
-    ((stream standard-output-recording-stream)
-     character text-style width height baseline)
-  (add-character-output-to-text-record
-   (stream-text-output-record stream text-style)
-   character text-style width height baseline))
+(defmethod stream-close-text-output-record ((stream standard-output-recording-stream))
+  (when-let ((record (stream-current-text-output-record stream)))
+    (setf (stream-current-text-output-record stream) nil)
+    #|record stream-current-cursor-position to (end-x record) - already done|#
```
Comment (nsrahmad) [inline Core/clim-basic/recording.lisp]: @dkochmanski is it a leftover note to self?

Comment (dkochmanski) [reply Core/clim-basic/recording.lisp]: no, this comment was already in this method. since I don't understand it I didn't remove it.

**Comment by dkochmanski** [comment]
pushed change removes comment I've made because change (that we don't have :around method but a proper specialization) and not eager drawing makes it unnecessary. We don't need to draw directly on medium anynome from standard output.

**Pull request merged**